src/components/Toast/index.tsx

```diff
@@ -29,6 +29,7 @@ const Toast: FC<ComponentProps> = ({children, className, type = 'warning'}) => {
         [`Toast--${type}`]: true,
         ...bemify(className, `--${type}`),
       })}
+      data-testid="Toast"
     >
       <Icon
         className={clsx('Toast__icon', {
```

Comment:
add a `data-testid` to this Icon: `data-testid="Toast__icon"`

add a `data-testid` to the text div below: `data-testid="Toast__text"`

Reply:
I think `data-testid` only works on normal HTML elements, not React Components because I get an error that it can't find an element with `data-testid=Toast__icon`

Reply:
@AbhayAysola if you `git pull` the latest changes, the `Icon` component has a `dataTestId` prop you can use

Reply:
Alright, thanks :)